app/models/mixins/miq_policy_mixin.rb

```diff
@@ -115,6 +115,10 @@ def check_policy_prevent_callback(*action, _status, _message, result)
     prevented ? _log.info(event.attributes["message"]) : send(*action)
   end
 
+  def policies_applied
+    get_policies.present?
+  end
```

Comment:
Note there is already a has_policies method defined that is essentially already this.

Reply:
Yes, actually there's a `has_policies?` method already there. But that calls `get_policies` and that method does a direct query on `Tag` for assignments and then a direct query on `MiqPolicy` or `MiqPolicySey` for each result.